src/controllers/user.js

```diff
@@ -1,20 +1,68 @@
+const { ObjectId } = require('mongoose').Types;
```

Comment:
Once you merge `main` branch you can override Perizat's temporary code changes, she has added comments for which parts are temporary.